src/components/HashButton.tsx

```diff
@@ -37,17 +37,18 @@ function getHashLinkStr(hash: string, type: HashType): string {
 }
 
 function HashLink(hash: string, type: HashType): JSX.Element {
+  const augmentToWithGlobalSearchParams = useAugmentToWithGlobalSearchParams();
+
   switch (type) {
     case HashType.ACCOUNT:
     case HashType.TRANSACTION:
       return (
-        <Link
-          component={RRD.Link}
-          to={getHashLinkStr(hash, type)}
+        <InternalLink
+          to={augmentToWithGlobalSearchParams(getHashLinkStr(hash, type))}
```

Comment:
do we still need `augmentToWithGlobalSearchParams` here since we're already using `InternalLink`?

Reply:
good find, nope